1.14/Dockerfile

```diff
@@ -1,11 +1,11 @@
 # vim:set ft=dockerfile:
 
-FROM cimg/base:2020.01
+FROM cimg/base:2020.03
 
 LABEL maintainer="Community & Partner Engineering Team <community-partner@circleci.com>"
 
-ENV GO_VERSION=1.14
-ENV GO_SHA=
+ENV GO_VERSION=1.14.1
+ENV GO_SHA=fake-sha
```

Comment:
What is the purpose of this?

Reply:
The first line sets the Go version and the second line is a placeholder for when SHA support is built.